subs2srs.lua

```diff
@@ -821,6 +827,7 @@ local main = (function()
         mp.add_key_binding("Ctrl+C", "mpvacious-copy-secondary-sub-to-clipboard", subs_observer.copy_current_secondary_to_clipboard)
         mp.add_key_binding("Ctrl+t", "mpvacious-autocopy-toggle", subs_observer.toggle_autocopy)
         mp.add_key_binding("Ctrl+g", "mpvacious-animated-snapshot-toggle", encoder.snapshot.toggle_animation)
+        mp.add_key_binding("Alt+m", "mpvacious-toggle-custom-sub-filter", subs_observer.toggle_custom_sub_filter)
```

Comment:
I think we don't need a key binding to change this. The user can just edit the config file.